### DISCUSSION TITLE
Another board to hack

### DISCUSSION THREAD

**Discussion by skiphansen** [xjtuecho/EBAZ4205#44] [General]
I've been playing with surplus [Pano Logic thin clients](https://github.com/tomverbeure/panologic-g2/wiki) for a couple of years now and I've been having a blast.  They are still readily available on ebay, if you are interested in the EBAZ4205 you might also take a look at the Panos as well.

I've bought a few EBZA4205's, but other than booting the stock FW and logging in I haven't done anything with them yet.
I'm very interested in add on boards to add at least a display and hopefully some pmod connectors.

**Comment by uwu64** [comment]
Too bad the prices for Pano Logic G2 modules are pretty high at the moment. Might pick some up when they come down a little

**Reply by skiphansen** [reply]
I stored an search on ebay to watch the "market", currently "reasonable" prices are from $15 to $25.  It's pot luck, the price doesn't see to depend on the generation.  International shipping can be the biggest cost by far.  It's interesting to see multiple auctions of still "new" in the box Pano's 8 years after the company when out of business..  They sure must have made a lot of them.

I'd suggest getting both a "first generation" and a Rev B of the second generation.  The first generation is a lesser device, but we (the hacker community) have been able to get USB working.  The Rev B is a much better device but no one (that I know of) has gotten USB running yet.  The second generation only as a USB PHY, not a dedicated controller like the first generation.
